presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUninstaller.java

```diff
@@ -40,7 +43,7 @@ public void runCommand()
             case RPM:
                 uninstallUsingRpm("presto-server-rpm", disableDependencyChecking, ignoreNotInstalled);
                 break;
-            case TARBALL:
+            case GZ:
```

Comment:
This should be `TARBALL` for now.

We can fix how that is handled later, when we add support for tarballs.